server/src/main/java/org/jboss/as/server/deployment/integration/Seam2Processor.java

```diff
@@ -78,19 +78,19 @@
     public static final ModuleIdentifier EXT_CONTENT_MODULE = ModuleIdentifier.create("org.jboss.integration.ext-content");
     public static final ModuleIdentifier VFS_MODULE = ModuleIdentifier.create("org.jboss.vfs");
 
-    private ServiceTarget serviceTarget; // service target from the service that added this dup
     private ResourceRoot seamIntResourceRoot;
 
-    public Seam2Processor(ServiceTarget serviceTarget) {
```

Comment:
This change isn't valid. I checked with Stuart and the JaxrsSpringProcessor one in full isn't either and needs to be reverted.

The service installed here is not scoped to the deployment, so using the deployment's ServiceTarget is not valid.

Perhaps this and the JaxrsSpringProcessor thing could be done differently, but I think we can make the OperationContextImpl classes work better. An attempt at this is at https://github.com/wildfly/wildfly-core/pull/2008